src/black/lines.py

```diff
@@ -103,7 +102,11 @@ def append_safe(self, leaf: Leaf, preformatted: bool = False) -> None:
         Raises ValueError when any `leaf` is appended after a standalone comment
         or when a standalone comment is not the first leaf on the line.
         """
-        if self.bracket_tracker.depth == 0:
+        if (
+            self.bracket_tracker.depth == 0
+            or self.bracket_tracker._for_loop_depths
```

Comment:
We should make these attributes public if we're using them outside the class.

Reply:
Agreed, wrapped in a function call